src/App.js

```diff
@@ -1,13 +1,73 @@
-import React from 'react';
+import React from "react";
+
+import ToDoForm from "./components/TodoComponents/TodoForm";
+import ToDoList from "./components/TodoComponents/TodoList";
+
+import "./App.css";
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+
+  state = {
+    inputData: "",
+    toDoArray: []
+  };
+
+  inputDataHandler = val => {
+    this.setState({
+      inputData: val
+    });
+  };
+
+  pushDataHandler = () => {
+    this.setState({
+      toDoArray: [
+        ...this.state.toDoArray,
+        {
+          task: this.state.inputData,
+          id: new Date().getTime(),
+          completed: false
+        }
```

Comment:
I would also recommend reseting the start for the inputData to "" here so that once you enter your item, the input is blank.